stdlib/test/builtin/test_repr.mojo

```diff
@@ -47,7 +47,24 @@ def test_none_representation():
     assert_equal(repr(None), "None")
 
 
+def test_escape_representation():
```

Comment:
Could you please move this to `test_string.mojo`, once you ensure it only uses `String.__repr__`?

Reply:
yup